modules/common/istio/variables.tf

```diff
@@ -1,2 +1,6 @@
 variable "namespace" {}
-variable "crd_waiter" {}
\ No newline at end of file
+variable "crd_waiter" {}
+variable "kiali_username" {
+  default = "admin"
+}
+variable "kiali_password" {}
```

Comment:
this should be auto-generated as a terraform random string, then stuffed in a secret